doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -180,6 +195,12 @@ value. The original state of :n:`@option` is *not* restored at the end of the
 module. Additionally, if unset in a file, :n:`@option` is reset to its
 default value when the file is `Require`-d.
 
+.. cmdv:: Export Unset @option.
+
+This command resets :n:`@option` to its default
+value. The original state of :n:`@option` is restored at the end of the
```

Comment:
That's not correct: options are sometimes on by default, and sometimes off by default.

Reply:
WDYM? The phrasing seems correct to me. It's an adaptation of what is written above already.

Reply:
Well, you are right. The text above is incorrect as well.
`Unset` does not reset to the default value. It unsets the option. Sometimes, options are set by default and must be unset to see a different result.

Reply:
That being said, your PR is not making the reference manual worse, so this remark shouldn't be blocking it.

Reply:
Why not fix it while we are at it?

Reply:
Btw this whole flag/option distinction seems very strange to me. We use the terminology option for flags in the rest of the manual.

Reply:
@ppedrot Could you fix the documentation ASAP please?

Reply:
@maximedenes I don't know what to write. Solely writing *unsets the option* sounds completely non-informative to me.

Reply:
`turns off`?

Reply:
It will be better that the current documentation, which is wrong, don't you think?